internal/controllers/core/liveupdate/reconciler.go

```diff
@@ -418,6 +419,17 @@ func (r *Reconciler) reconcileKubernetesResource(ctx context.Context, monitor *m
 		return false, err
 	}
 
+	if selector.ImageMap != "" {
```

Comment:
can you add some tests for the ImageMap case? (tbh, it feels like the default test fixture should now use image-map-based selectors)

Reply:
Good call! I changed the fixture default to use the `ImageMapName` for a selector in b1a96d7a5. (I also made the `ImageMap` object use a different `ImageFromCluster` to hopefully increase the odds of catching local registry bugs)